server/src/cg/group4/server/database/ResponseHandler.java

```diff
@@ -1,5 +1,13 @@
 package cg.group4.server.database;
 
```

Comment:
It was to fix the Android SQLite server. SQLDroid wouldn't function with multiple connections. Decided to have 1 connection that has many statements.